src/app/routes/gateway.js

```diff
@@ -109,4 +109,17 @@ module.exports = (Router, Service) => {
       });
   });
 
+  Router.get('/gateway/checkout/session', basicAuth, (req, res) => {
+    const sessionId = req.query.sessionId;
+
+    return Service.Stripe.findSessionById(sessionId)
```

Comment:
use `async-await`, we have a catch-all handler,`try-catch` is not required anymore. Example [here](https://github.com/internxt/drive-server/blob/33ad1ac4a7b547f6d89d6aa419628850c915f89f/src/app/routes/storage.js#L256).